warehouse/api/openapi.yaml

```diff
@@ -0,0 +1,112 @@
+openapi: 3.0.0
```

Comment:
Supports OpenAPI v3.0.0 - not 3.1.0 yet, since the underlying library hasn't been updated yet.
Since it's brand new, we can proceed if we like it, and we're not bound to any specific schema yet, and I don't think we're using too many things that would change with the new version.